packages/ci/src/utils/source.ts

```diff
@@ -42,7 +40,7 @@ export async function createSourceBlob(ref: any, command: Command) {
     if (await urlExists(archiveLink.archive_link)) {
       return archiveLink.archive_link
     }
-  } catch (ex) { command.error(ex) }
+  } catch { }
```

Comment:
Unfamiliar with this util, why catch and do nothing?

Reply:
if the intention is to swallow any error, it should at least go to `ux.debug()`

Reply:
I added a comment and logged it to `ux.debug()`. The situation is that you can run CI for commits that are not pushed to the remote. We use git to package it up and send it directly to heroku to run ci against.